src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -0,0 +1,320 @@
+<?php
+
+namespace Illuminate\Foundation\Testing;
+
+use Closure;
+use Illuminate\Support\Str;
+use Illuminate\Http\Response;
+use Illuminate\Contracts\View\View;
+use PHPUnit_Framework_Assert as PHPUnit;
+
+class TestResponse extends Response
+{
+    /**
+     * Convert the given response into a TestResponse.
+     *
+     * @param  \Illuminate\Http\Response  $response
+     * @return static
+     */
+    public static function fromBaseResponse($response)
+    {
+        $testResponse = new static(
+            $response->getContent(), $response->status()
+        );
+
+        $testResponse->headers = $response->headers;
+
+        if (isset($response->original)) {
+            $testResponse->original = $response->original;
+        }
+
+        if (isset($response->exception)) {
+            $testResponse->exception = $response->exception;
+        }
+
+        return $testResponse;
+    }
+
+    /**
+     * Assert that the response has an OK status code.
+     *
+     * @return void
+     */
+    public function assertHasStatus($status)
+    {
+        $actual = $this->getStatusCode();
+
+        PHPUnit::assertTrue($this->isOk(), "Expected status code 200, got {$actual}.");
```

Comment:
1st parameter provide `$status`, but assertion only check for `Response::isOk()`.